README.md

```diff
@@ -188,7 +188,7 @@ private fun showPaymentCreatorActivity() {
     intent.putExtra(OmiseActivity.EXTRA_AMOUNT, 150000L)
     intent.putExtra(OmiseActivity.EXTRA_CURRENCY, "thb")
 
-    // you can retrieve your account's capabilities through the SDK (will be explained in a different section)
+    // you can retrieve your account's capabilities through the SDK (will be explained in the below)
```

Comment:
Isn't here something missing? I think it should be `will be explained in the section below` or `will be explained below`.

Reply:
@tobias90 Thanks, I has updated in this commit [bc0b5ac](https://github.com/omise/omise-android/pull/132/commits/bc0b5ac40517334d66776bb68008c7ccd335e533)